windows-active-directory-mixin/README.md

```diff
@@ -0,0 +1,111 @@
+# Windows Active Directory mixin
+
+The Windows Active Directory mixin is a set of configurable Grafana dashboards and alerts.
+
+The Windows Active Directory mixin contains the following dashboards:
+
+- Windows Active Directory overview
+- Windows Active Directory logs
+
+and the following alerts:
+- alertsCriticalHighReplicationIssues 
```

Comment:
I believe these should be the names of the alerts and not the thresholds

Reply:
^^^